### ISSUE TITLE
Include hessians function

### ISSUE THREAD

**Issue by MSolodarenko** [malmaud/TensorFlow.jl#417]
Why does "gradients" function is included, but "hessians" is not?

Can this function (https://www.tensorflow.org/api_docs/python/tf/hessians) be added as well?

Thanks!

**Comment by oxinabox** [comment]
`gradients` is included because it is required to make the optimizers for training.

Hessians would be nice to have. I've wanted it before.
(Like gradients) there is no operator for it in libtensorflow.
But it can be defined using `gradients` (and some other things like `unpack` or slicing etc).

I personally do not have time to add it right now.
I'ld happily review a PR.
